runs/2020/pyladies-ostrava-podzim/link.yml

```diff
@@ -0,0 +1,2 @@
+repo: https://github.com/pyvec/naucse.python.cz/
```

Comment:
```suggestion
repo: https://github.com/PyLadiesCZ/naucse.python.cz/
```

Reply:
❤ tady jsem si nebyla jistá, jak to funguje